### PR TITLE
[kubevirt ssp operator] add testing records for branch release-v0.13 for ssp operator

### PR DESCRIPTION
**Pull request by ksimon1**
add testing records for branch release-v0.13 for ssp operator
Signed-off-by: Karel Šimon <ksimon@redhat.com>

### CODE REVIEW

File: clusters/app.ci/supplemental-ci-images/libguestfs-tools.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+apiVersion: image.openshift.io/v1
```
Comment (petr-muller) [inline clusters/app.ci/supplemental-ci-images/libguestfs-tools.yaml:1]: No. Please follow the process documented in https://docs.ci.openshift.org/docs/how-tos/external-images/ to see how to import external images to CI cluster.

Comment (petr-muller) [reply clusters/app.ci/supplemental-ci-images/libguestfs-tools.yaml:1]: Seems like you already did that in https://github.com/openshift/release/pull/25423#issuecomment-1016906942 so this part of this PR can simply be removed and then you do not need TP to approve.

/uncc
/unassign @smg247

Comment (ksimon1) [reply clusters/app.ci/supplemental-ci-images/libguestfs-tools.yaml:1]: Sorry, this is leftover from another PR. I will remove it.

**Pull request merged**